modules/updating-eus-to-eus-upgrade-cli.adoc

```diff
@@ -0,0 +1,144 @@
+// Module included in the following assemblies:
+//
+// * updating/preparing-eus-eus-upgrade.adoc
+
+:_content-type: PROCEDURE
+[id="updating-eus-to-eus-upgrade-cli_{context}"]
+= EUS-to-EUS update using the CLI
+
+.Prerequisites
+
+* Update OpenShift CLI `oc` to the target version before each update.
+
+[IMPORTANT]
+====
+It is highly discouraged to skip this prerequisite. If the OpenShift CLI `oc` is not updated to the target version before your update, unexpected issues may occur.
+====
+
+.Procedure
+
+. Using the Administrator perspective on the web console, update any Operator Lifecycle Manager (OLM) Operators to the versions that are compatible with the version you are updating to. You can find more information on how to perform this action on the link:https://docs.openshift.com/container-platform/4.12/operators/admin/olm-upgrading-operators.adoc[Updating installed Operators] page.
+
+. Verify that all machine config pools display a status of `UPDATED` and that no machine config pool displays a status of `UPDATING`.
+To view the status of all machine config pools, run the following command:
++
+[source,terminal]
+----
+$ oc get mcp
+----
++
+.Example output
++
+[source,terminal]
+----
+NAME     CONFIG                                         	UPDATED   UPDATING
+master   rendered-master-ecbb9582781c1091e1c9f19d50cf836c       True  	  False
+worker   rendered-worker-00a3f0c68ae94e747193156b491553d5       True  	  False
+----
+
+. Change to the `eus-4.12` channel by running the following command:
++
+[source,terminal]
+----
+$ oc adm upgrade channel eus-4.12
+----
++
+[NOTE]
+====
+
+If you receive an error message indicating that `eus-4.12` is not one of the
+available channels, this indicates that Red Hat is still rolling out 4.10 to 4.12 EUS updates.
+This rollout process generally takes 45-90 days starting at the GA date.
+====
++
+
+. Pause all worker machine pools except for the master pool by running the following command:
++
+[NOTE]
```

Comment:
nit: I would be placing the command above the note, as the rest of them

Reply:
good call :)